browser-raw.js

```diff
@@ -76,9 +76,9 @@ function flush() {
 
 // Safari 6 and 6.1 for desktop, iPad, and iPhone are the only browsers that
 // have WebKitMutationObserver but not un-prefixed MutationObserver.
-// Must use `global` instead of `window` to work in both frames and web
-// workers. `global` is a provision of Browserify, Mr, Mrs, or Mop.
-var BrowserMutationObserver = global.MutationObserver || global.WebKitMutationObserver;
+var BrowserMutationObserver = global ?
```

Comment:
To function properly in workers and windows, we probably can use use `self`. Needs to be tested in both environments.